layouts/community/ortho_4x12/neo2marin/keymap.c

```diff
@@ -0,0 +1,173 @@
+/* Copyright 2018-2022 TheAmarin (@theamarin)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+ 
+#include QMK_KEYBOARD_H
+#include "keymap_german.h"
+
+enum custom_keycodes {
+  NEO2 = SAFE_RANGE,
+  QWERTZ,
+};
+
+#define L_NEO    0 // NEO2
+#define L_QWZ    1 // QWERTZ
+#define L_COD    2 // NEO2 Layer 3 (Coding)
+#define L_COD_AT 3 // NEO2 Layer 3 (Coding with AT)
+#define L_MOV    4 // NEO2 Layer 4 (Movements)
+#define L_FUN    5 // Function and media keys
```

Comment:
We heavily recommend enums for layer naming, as they automatically increment, which makes long term management a lot easier, and less prone to accidental bloat due to empty layers. 

```suggestion
enum custom_layers {
    L_NEO,
    L_QWZ,
    L_COD,
    L_COD_AT,
    L_MOV,
    L_FUN,
};
```

Reply:
:+1: